features/step_definitions/tag_steps.rb

```diff
@@ -19,42 +19,48 @@
              ArchiveConfig.RATING_MATURE_TAG_NAME,
              ArchiveConfig.RATING_EXPLICIT_TAG_NAME]
   ratings.each do |rating|
-    Rating.find_or_create_by_name_and_canonical(rating, true)
+    Rating.find_or_create_by(name: rating, canonical: true)
   end
 end
 
+Given(/^an adult canonical rating exists with name: "([^"]*)"$/) do |rating|
+  Rating.find_or_create_by(name: rating, canonical: true, adult: true)
+end
+
 Given /^the basic warnings exist$/ do
-  Warning.find_or_create_by_name_and_canonical("No Archive Warnings Apply", true)
-  Warning.find_or_create_by_name_and_canonical("Choose Not To Use Archive Warnings", true)
+  Warning.find_or_create_by_name("No Archive Warnings Apply").update(canonical: true)
```

Comment:
Warning and Category inherit from `Tag` which has a custom defined `find_or_create_by_name` method. Calling `find_by(name:` directly skips this method (and some important logic). I was keeping all Tag children using the custom method. In this case, it might be simply using the find_or_create logic. Perhaps that method should be renamed to avoid confusion with Rails provided finders?